src/dowel/csv_output.py

```diff
@@ -48,7 +50,35 @@ def record(self, data, prefix=''):
                            'Did you change key sets after your first '
                            'logger.log(TabularInput)?'.format(
                                set(self._fieldnames), set(to_csv.keys())))
-
+                
```

Comment:
The warning above this line is no longer necessary, since the issue is now handled. Can you delete it and any other unused code?

Reply:
Updated